lib/global_id/uri/gid.rb

```diff
@@ -154,6 +157,13 @@ def validate_component(component)
           "Expected a URI like gid://app/Person/1234: #{inspect}"
       end
 
+      def validate_model_id(model_id)
+        return model_id unless model_id.blank?
+
+        raise MissingModelId, "Unable to create a global id for #{model_name} " \
```

Comment:
I'm not sure what you mean exactly. Isn't `model_name` guaranteed to be present at this point since `validate_component(model_name)` would have already failed if `model_name` were blank?